contrib/construct_dictionary.py

```diff
@@ -9,20 +9,16 @@
 # by tools to interpret user input
 #
 
-from __future__ import print_function
```

Comment:
I'm pretty sure the config will now allow us to use Python 2.x, which means we need that "future" import, yes?

Reply:
Nope -- we don't need this for `print()`.  I've tested the script with Python 2.6.6.